src/Web/Sfa.Das.Sas.Shared.Components/ViewComponents/TrainingProvider/ContactDetails/TrainingProviderContactDetailsViewComponent.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Sfa.Das.Sas.ApplicationServices.Models;
+using Sfa.Das.Sas.Shared.Components.Orchestrators;
+using Sfa.Das.Sas.Shared.Components.ViewComponents.ApprenticeshipDetails;
+using Sfa.Das.Sas.Shared.Components.ViewComponents.TrainingProvider.Search;
```

Comment:
Unused usings

Reply:
have refactored view components to make this a view of the details component rather than a separate view component